test/host/hls/test.cpp

```diff
@@ -100,7 +100,7 @@ std::unique_ptr<ACCL::ACCL> test_vadd_put(options_t options) {
 
         src_bo.write(src);
         src_bo.sync(XCL_BO_SYNC_BO_TO_DEVICE);
-        auto run = vadd_ip(src_bo, dst_bo, options.count, (rank+1)%size, accl->get_communicator_addr(), 
```

Comment:
revert cosmetic changes (spaces, empty lines added or deleted) here and everywhere. the PR is very large, we should condense it to the minimum viable change

Reply:
I'll revert the unchanged files, but I don't think it's worth the effort of trying to get back the trailing whitespace on files that have been changed.